Emby.Server.Implementations/Library/DefaultPasswordResetProvider.cs

```diff
@@ -0,0 +1,118 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Threading.Tasks;
+using MediaBrowser.Common.Configuration;
+using MediaBrowser.Common.Extensions;
+using MediaBrowser.Controller;
+using MediaBrowser.Controller.Authentication;
+using MediaBrowser.Controller.Configuration;
+using MediaBrowser.Controller.Entities;
+using MediaBrowser.Controller.Library;
+using MediaBrowser.Model.Serialization;
+using MediaBrowser.Model.Users;
+using ServiceStack;
```

Comment:
What's this used for?

Reply:
I thought I needed service stack to properly get the dependency injection going so the constructor for it works. I'll double back on this real quick and confirm.

Reply:
Lol nope, it got autoadded because it provides a ToArray() extension method like Linq. replaced.